shardingsphere-scaling/shardingsphere-scaling-core/src/main/java/org/apache/shardingsphere/scaling/core/execute/executor/importer/AbstractJDBCImporter.java

```diff
@@ -123,12 +126,24 @@ private void executeInsert(final Connection connection, final DataRecord record)
         ps.setQueryTimeout(30);
         try {
             for (int i = 0; i < record.getColumnCount(); i++) {
-                ps.setObject(i + 1, record.getColumn(i).getValue());
+                SQLType sqlType = extractTargetSQLType(record.getColumn(i).getValue());
+                if (sqlType != null) {
+                    ps.setObject(i + 1, record.getColumn(i).getValue(), sqlType);
+                } else {
+                    ps.setObject(i + 1, record.getColumn(i).getValue());
+                }
             }
             ps.execute();
         } catch (SQLIntegrityConstraintViolationException ignored) {
         }
     }
+
+    private SQLType extractTargetSQLType(final Object recordValue) {
+        if (recordValue instanceof BigInteger) {
+            return JDBCType.BIGINT;
+        }
+        return null;
+    }
```

Comment:
Only process Bigint is not eough, we'd better cover all type.

Reply:
I tried some types , currently just Bigint will cause that issue.I will test all types later , so that's why I make that check as a method, may add more types in `extractTargetSQLType `